aspnetcore/fundamentals/http-context.md

```diff
@@ -88,6 +88,9 @@ public void ConfigureServices(IServiceCollection services)
 }
 ```
 
+> [!NOTE]
+> <xref:Microsoft.Extensions.DependencyInjection.IdentityServiceCollectionExtensions.AddIdentity*> provides <xref:Microsoft.Extensions.DependencyInjection.HttpServiceCollectionExtensions.AddHttpContextAccessor*> registration. An additional explicit registration can be added without generating an error or performance penalty.
```

Comment:
The "provides AddHttpContextAccessor registration" part of the first sentence sounds odd to me. I suggest replacing the following:

```
provides <xref:Microsoft.Extensions.DependencyInjection.HttpServiceCollectionExtensions.AddHttpContextAccessor*> registration.
```

with this:

```
registers <xref:Microsoft.AspNetCore.Http.IHttpContextAccessor>.
```

Thoughts on this, @davidfowl?